test/test_value_ranges.py

```diff
@@ -119,15 +122,45 @@ def ceil(x):
         return math.ceil(x)
 
 
+def valid_unary(fn, v):
```

Comment:
we could also run the reference first and see if it throws, then skip

Reply:
Still good to avoid trying to eval something silly like 2 ** 10000 which sympy will do symbolically lol